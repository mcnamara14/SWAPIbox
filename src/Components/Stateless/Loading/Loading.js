import React from 'react';
import './Loading.css';
import loadingSound from './loading.wav';

const Header = (props) => {
  
  return (
    <div className="loading">
      <audio src={loadingSound} autoPlay loop></audio>
      <img src={require('./loading.gif')} height="400" alt={'Loading gif'}/>
    </div>
  );
}

export default Header;
