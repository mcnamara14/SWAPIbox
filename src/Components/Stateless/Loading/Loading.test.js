import React from 'react';
import ReactDOM from 'react-dom';
import Loading from './Loading';
import { shallow } from 'enzyme';

describe('Card', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Loading />)
  })

  it('matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  });

});
