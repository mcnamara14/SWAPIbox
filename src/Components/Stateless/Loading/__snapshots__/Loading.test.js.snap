// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Loading />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <audio
          autoPlay={true}
          loop={true}
          src="loading.wav"
/>,
        <img
          alt="Loading gif"
          height="400"
          src="loading.gif"
/>,
      ],
      "className": "loading",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "autoPlay": true,
          "loop": true,
          "src": "loading.wav",
        },
        "ref": null,
        "rendered": null,
        "type": "audio",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "Loading gif",
          "height": "400",
          "src": "loading.gif",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <audio
            autoPlay={true}
            loop={true}
            src="loading.wav"
/>,
          <img
            alt="Loading gif"
            height="400"
            src="loading.gif"
/>,
        ],
        "className": "loading",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "autoPlay": true,
            "loop": true,
            "src": "loading.wav",
          },
          "ref": null,
          "rendered": null,
          "type": "audio",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "Loading gif",
            "height": "400",
            "src": "loading.gif",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
